data/json/items/comestibles/meat_dishes.json

```diff
@@ -874,7 +874,7 @@
     "comestible_type": "FOOD",
     "symbol": "%",
     "calories": 400,
-    "description": "Bright white chicken-paste.",
+    "description": "Off-white chicken-paste in a can!",
```

Comment:
Generally it's not a good idea to state that an item is inside a certain container in the item description, as the player could remove item from said container, but description wouldn't be changed.

Reply:
This description was changed to better fit the description of canned salmon ("Bright pink fish-paste in a can!"), which AFAIK has always been that item's description. I see what you're saying though. The question is whether to account for the unlikely motivation of the player to purposely remove canned perishable foods from cans without immediately eating it, and if yes, how that should be done. Ending the description with "...from a can!" instead, perhaps?